Prace_domowe/Praca_domowa1/Grupa4/Marcin Wilk/PD1.ipynb

```diff
@@ -0,0 +1,4755 @@
+{
```

Comment:
<ol><li>ta wizualizacja jest słaba \:( Pozioma byłaby lepsze</li><li>Akurat lasy lubią&nbsp;soboty, wtedy jest więcej pożarów -&gt; wykres liczba pożarów vs. dzień tygodnia</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Tu napisałem trochę nieprawdę. Faktycznie, jeśli rozkład jest podobny dla każdego dnia tygodnia, to można wywalić tę zmienną, zwłaszcza jak budujemy wstępny, prosty model. Natomiast porównanie rozkładów powinno być porządniejsze niż wyplotowania wykresów. Zauważ, że nie mają nawet wspólnej osi x. </p>